store/index.tsx

```diff
@@ -38,6 +38,7 @@ export const AppStateProvider: React.FC<{ children: React.ReactNode }> = ({ chil
     })
 
     useEffect(() => {
+        const hash = window.location.hash.slice(1);
```

Comment:
why we need this

Reply:
that's a different thing entirely, i'm trying to fix the hash issue.

Reply:
Let's keep this out of this PR. You can use a different branch to do this